src/pages/MyNetwork/MyNetwork.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../styles/Breakpoints.scss';
+@import '../../styles/Colors.scss';
+
+.myNetwork {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $linkedinBackgroundMainTheme;
+  min-height: 100vh;
+
+  &__container {
+    margin: 20px 0;
+    width: 60vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: $white;
+    padding: 10px;
+    border: 1px solid $lightGray;
+  }
+
+  &__buttons {
+    min-width: 72%;
+    margin: 20px 0;
+    display: flex;
+    flex-direction: column;
+    border-radius: 5px;
+    background-color: $white;
+
+    @media only screen and (min-width: $laptop) {
+      flex-direction: row;
+    }
+  }
+
+  &__button {
+    flex: 1;
+    border: none !important;
+    border-radius: 0 !important;
```

Comment:
important is necessarry?

Reply:
![image](https://user-images.githubusercontent.com/87152087/128005899-21437090-4450-49ca-b4b6-ce82a5632d8c.png)
 yes important is needed in this case because it overwrites styles from material ui